readthedocs/privacy/backend.py

```diff
@@ -122,29 +167,17 @@ def private(self, user=None, project=None, only_active=True):
     def api(self, user=None):
         return self.public(user, only_active=False)
 
-    def create_stable(self, **kwargs):
-        defaults = {
-            'slug': STABLE,
-            'verbose_name': STABLE_VERBOSE_NAME,
-            'machine': True,
-            'active': True,
-            'identifier': STABLE,
-            'type': TAG,
-        }
-        defaults.update(kwargs)
-        return self.create(**defaults)
+    def for_project(self, project):
```

Comment:
So, the serving bug would exist here, but because we are only serving public projects, it's can't be reproduced. The bug is most noticeable when there is a mismatch in privacy levels on versions.